config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-ci.yaml

```diff
@@ -121,3 +121,47 @@ periodics:
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-gcp
     testgrid-tab-name: e2e-tests-stable
+- name: ci-cluster-api-provider-gcp-make-conformance-stable-k8s-ci-artifacts
+  interval: 3h
+  labels:
+    preset-service-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191012-482f444-master
+        env:
+          - name: "CAPI_BRANCH"
+            value: "stable"
+          - name: "CABPK_BRANCH"
+            value: "stable"
+        args:
+          - "--repo=k8s.io/kubernetes=master"
+          - "--repo=sigs.k8s.io/cluster-api-provider-gcp=master"
```

Comment:
Should we cut and target a "stable" release branch here? Testing TIP/TIP seems like we are asking for issues when troubleshooting failures.

Reply:
yea, possibly. let me get the harness working and we can revisit the matrix.

Reply:
ok, works for me, but we should probably at least restrict cluster-api to `release-0.2`, otherwise the breaking v1alpha3 changes are likely to come into play.

Reply:
the CAPI_BRANCH=stable takes care of using a pinned release. so we should be fine (and i'll remove the unnecessary repos later as well)